linux/kernel/building.md

```diff
@@ -208,6 +251,18 @@ sudo umount mnt/fat32
 sudo umount mnt/ext4
```

Comment:
I wonder if it might make sense to move the `sudo umount mnt/ext4` lines immediately after the `modules_install` lines? :shrug:

Reply:
While i like that idea, i think that may be outside the scope of this PR, if you fell otherwise I would be glad to change it.

Reply:
I agree - let's not make the perfect the enemy of the good.